packages/verification-vue-component/src/components/Verification/item/cards/SigningCard.vue

```diff
@@ -82,19 +82,37 @@
                         </template>
                     </DataPanel>
                 </div>
+                <div v-if="retractEvents.length > 0 && retractEvents[0].on_blockchain" class="section retraction-hash">
+                    <div class="label">{{ _$t('verification.result.meta.retractionTransaction') }}</div>
+
+                    <DataPanel :key="`expert-${retractEvents[0].ref}`">
+                        <template #header>
+                            <a :href="`https://${net}/tx/${retractEvents[0].on_blockchain.tx_hash}`"
+                               target="_blank">{{ retractEvents[0].on_blockchain.tx_hash }}</a>
+                        </template>
+                    </DataPanel>
+                </div>
             </div>
         </template>
         <template v-else #body>
             <ResultDetail verification-mode="signing"
                           :document-revoked="revokeEvents.length > 0"
                           :document-revocation-in-progress="revocationInProgress"
                           :document-revocation-date="revocationDate"
+                          :document-retracted="retractEvents.length > 0"
+                          :document-retraction-in-progress="retractionInProgress"
+                          :document-retraction-date="retractionDate"
                           :signatures-in-progress="signaturesInProgress"
                           :has-unverified-signer="hasUnverifiedSigner"
                           :has-verified-signer="hasVerifiedSigner"
                           :signer-count="signEvents.length"/>
 
             <div class="verification-info">
+                <div v-if="retractEvents.length > 0" class="section retraction-note">
+                    <span class="note-label" v-html="_$t('verification.result.meta.retractionNote')"/>
+                    <span class="note-value" v-html="retractionNote" />
+                </div>
```

Comment:
I think this isn't really thought through in the layout, how should it look like if we also have a note from the initiator? Does the retracted note override the initial note? Are both displayed?

Reply:
I didn't have the multiple notes in mind when implementing it and I don't think that's something we have to tackle now here but you are still right. @stephalba What do you think? Are you aware of it?

Reply:
Just discussed it with steph, it's fine as it is now because it's not even sure how and when other notes are coming.